src/editor/Gui.cpp

```diff
@@ -2274,17 +2269,21 @@ void Gui::drawSettings() {
 
 			if (ImGui::Button("Apply Changes")) {
 				g_settings.gamedir = string(gamedir);
-
-				g_settings.fgdPaths.clear();
-				for (int i = 0; i < numFgds; i++) {
-					g_settings.fgdPaths.push_back(fgdPaths[i]);
+				if (g_settings.gamedir.size() == 0)
+				{
+					g_settings.gamedir = "/";
+					gamedir[0] = '/';
+				}
+				else
+				{
+					if (g_settings.gamedir[g_settings.gamedir.size() - 1] != '\\' &&
+						g_settings.gamedir[g_settings.gamedir.size() - 1] != '/')
+					{
+						g_settings.gamedir += '/';
+						gamedir[g_settings.gamedir.size()] = '/';
+						gamedir[g_settings.gamedir.size()] = '\0';
 				}

```

Comment:
anyone who updates is gonna need to click this apply button to fix their paths, because the ending slash wasn't allowed before. Ideally the slash should be added as needed, whenever the gamedir is used. For now I'd just remove this.

Reply:
As you wish, it seems to so better))

Reply:
I see gamedir not used in code, because all fgdPaths using non relativel path.
May be need fix it?

Reply:
> anyone who updates is gonna need to click this apply button to fix their paths, because the ending slash wasn't allowed before. Ideally the slash should be added as needed, whenever the gamedir is used. For now I'd just remove this.

@wootguy I see gamedir not used in code, because all fgdPaths using non relativel path.
May be need fix it?

Reply:
Updated. Fixed.
![xSVYX44 1](https://user-images.githubusercontent.com/62130676/108463568-ad668900-728f-11eb-9990-4bd187d5630d.png)